OpenGL/OpenGL/src/Application.cpp

```diff
@@ -67,17 +67,21 @@ int main(int argc, char* argv[]) {
     Viewport window(WIDTH, HEIGHT);
     Shader shader("src/shader/simple.vert", "src/shader/simple.frag");
     Terrain terrain;
-    Hitbox character;
+    Hitbox character({ 0.0f,32.0f,0.0f });
     ShadowMap shadows(1024);
     Loader loader;
-    Raycast caster(500.f);
+    Raycast caster(100.f);
     std::unique_ptr<Mesh> targetBlock = makeTargetBlock();
 
     GLuint textureID = loader.loadTexture("Texture_atlas");
 
-    window.camera.setLookAt({100, 100, 100}, {0, 0, 0});
+    window.camera.goToHead(character);
```

Comment:
can you put it the other way around ? `character.moveCamera(window.camera)`
I feel like Character depends more on a camera than a camera depends on a Character, so it's a shame for camera to be dependent on Character.